packages/plugin-list/src/ListHtmlDomRenderer.ts

```diff
@@ -10,7 +11,12 @@ export class ListHtmlDomRenderer extends AbstractRenderer<Node[]> {
     async render(node: ListNode): Promise<Node[]> {
         const tag = node.listType === ListType.ORDERED ? 'OL' : 'UL';
         const domListNode = document.createElement(tag);
-        this.engine.renderAttributes(node.attributes, domListNode);
+        const attributes = node.modifiers.find(
+            modifier => modifier.constructor.name === 'Attributes',
+        ) as Attributes;
```

Comment:
?

Reply:
Artifact from when I didn't prioritize "direct" instances in `get`.